src/Compilers/CSharp/Test/EndToEnd/EndToEndTests.cs

```diff
@@ -172,7 +172,7 @@ void tryCompileDeepFluentCalls(int depth)
 
         // This test is a canary attempting to make sure that we don't regress the # of fluent calls that 
         // the compiler can handle. 
-        [Fact, WorkItem("https://devdiv.visualstudio.com/DevDiv/_workitems/edit/1874763")]
```

Comment:
Instead of skipping the test entirely, consider reducing the limits for the legs that have problems.

Reply:
From looking at Cyrus's logs, it appeared to be more than one leg. In order to unblock, I opted for a blanket skip until we have time to investigate which legs are failing.

Reply:
Sounds good